_model-utils/getCollection.md

```diff
@@ -0,0 +1,14 @@
+---
+header: getCollection
+example: ModelUtils.getCollection(path, [models], [options], [callback])
+---
+
+Factory method for collections. Given the path of a collection, will return an instance of that collection. This will also add the `models` to that collection, and set any `options` to the collection.  `callback` allows the method to be called asynchronously. 
+
+**path** The path to the collection, for example: `my_collection`, would return an instance of a collection stored in `app/collections/my_collection.js`
```

Comment:
The path to the collection.  For example, `my_collection` will return an instance of a collection stored in `app/collections/my_collection.js`